load/PreValidate.py

```diff
@@ -40,6 +40,25 @@ def toString(self):
 
 class PreValidate:
 
+	def getStockHumbersFromDirectory(directory, bucket = "etl-development-input"):
```

Comment:
proposed new name: getStockNumbersFromFile

Reply:
The method name has been updated.